src/titiler/core/titiler/core/factory.py

```diff
@@ -862,13 +862,16 @@ def wmts(
                 supported_crs = f"EPSG:{tms.crs.to_epsg()}"
             else:
                 supported_crs = tms.crs.srs
+            
+            bounds_crs = tms.crs.geographic_crs.srs or "urn:ogc:def:crs:OGC:2:84"
```

Comment:
```suggestion
            bounds_crs = tms.crs.geographic_crs.srs or "urn:ogc:def:crs:OGC:2:84"
```

did you meant `geodetic_crs` instead of `geographic_crs` ? or maybe `tms.geographic_crs` ?

the issue is that the `srs` is not a `uri` 

```
In [4]: pyproj.CRS.from_epsg(4326).geodetic_crs.srs
Out[4]: 'GEOGCRS["WGS 84",ENSEMBLE["World Geodetic System 1984 ensemble",MEMBER["World Geodetic System 1984 (Transit)"],MEMBER["World Geodetic System 1984 (G730)"],MEMBER["World Geodetic System 1984 (G873)"],MEMBER["World Geodetic System 1984 (G1150)"],MEMBER["World Geodetic System 1984 (G1674)"],MEMBER["World Geodetic System 1984 (G1762)"],MEMBER["World Geodetic System 1984 (G2139)"],ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1]],ENSEMBLEACCURACY[2.0]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433]],CS[ellipsoidal,2],AXIS["geodetic latitude (Lat)",north,ORDER[1],ANGLEUNIT["degree",0.0174532925199433]],AXIS["geodetic longitude (Lon)",east,ORDER[2],ANGLEUNIT["degree",0.0174532925199433]],USAGE[SCOPE["Horizontal component of 3D system."],AREA["World."],BBOX[-90,-180,90,180]],ID["EPSG",4326]]'
```

Reply:
ahh yes `tms.geographic_crs` is the right call here. I think I also confirmed locally that the bound's crs must be a urn so will look to change that.